site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'How Photoshop solved working with files larger than can fit into memory'
+subhead: >
+  Learn how Adobe managed to let users edit even the biggest files on the web version of its iconic Photoshop app.
+date: 2023-03-21
+# updated: 2023-03-21
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/pYiISdhsJe3oEL871Dqp.png
```

Comment:
I suggest adding a different thumbnail (using `thumbnail:`) here, if you look at the listing preview the image is not great. https://pr-5717-static-dot-dcc-staging.uc.r.appspot.com/blog/

Reply:
That looks better, please merge this when you are ready.

Reply:
Thanks for the love for the detail. I didn't notice this because I only ever saw the page itself, not the overview.